controller/deploy/operator/contribute/update-contributions.sh

```
@@ -0,0 +1,76 @@
+  user_fork="git@github.com:${GITHUB_USER}/${repo_name}.git"
+
+  if [ ! -d "$dir" ]; then
+    echo "Cloning $url into $dir ..."
+    git clone "$url" "$dir"
+  else
+    echo "Using existing directory: $dir"
+  fi
+
+  # Add or update the "user" remote for the fork
+  (
+    cd "$dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)